packages/core/src/editor.tsx

```diff
@@ -23,6 +25,7 @@ export function Editor<K extends string = string>({
     reducer,
     createInitialState(plugins, defaultPlugin)
   )
+  const [clipboard, setClipboard] = React.useState<PluginState[]>([])
```

Comment:
Any reason why this isn't part of the store?

Reply:
it is, but it's not part of the state if that is what you mean? I could probably also move it there. Just thought it doesn't need to be because it doesn't affect the history or documents at all. Could also be easier to move it to a shared Context for all Editors, if we have this seperate.